app/services/follows/check_cached.rb

```diff
@@ -4,6 +4,10 @@ def self.call(follower, followable_type, followable_id)
       new(follower, followable_type, followable_id).call
     end
 
+    def self.delete(follower, followable_type, followable_id)
```

Comment:
Normally we'd make this a separate service, e.g. `Follows::DeleteCached`. The general idea is that one service is responsible for one domain interaction.

Reply:
That makes sense - thanks!